### PR TITLE
fix: move default api version to a constant

### PR DESCRIPTION
**Pull request by peternhale**
@W-16407766@
remove the constants.json file in favor of exported const for api version

### CODE REVIEW

File: test/generators/baseGenerator.test.ts [hunk 1]
```diff
@@ -60,12 +60,3 @@ describe('BaseGenerator', () => {
     expect(validateOptionsStub.calledOnce).to.be.true;
   });
 });
-
-describe('getDefaultApiVersion', () => {
```
Comment (CristinaCanizales) [inline test/generators/baseGenerator.test.ts:64]: Why delete this test?

File: yarn.lock
Comment (CristiCanizales) [inline yarn.lock:1]: was this file supposed to be included? I don't see any changes in package.json

File: test/generators/baseGenerator.test.ts (hunk 1 above)
Comment (peternhale) [reply test/generators/baseGenerator.test.ts:64]: It is just a constant now, no magic to get it loaded from json.

File: yarn.lock
Comment (peternhale) [reply yarn.lock:1]: The change is due to a sinon module change, indirect dependency for tests

**Review by CristiCanizales — APPROVED** [review]

**Pull request merged**